content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors (ACEis) and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.
+Data from some animal models suggest several but not all ACEis and several ARBs increase ACE2 expression in cells of some organs.
+Clinical studies have not established that ACE2 expression is increased in humans treated with these medications.
+While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACEis or ARBs and outcomes in patients with COVID-19.
+Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACEi at time of admission.[@doi:10.1056/NEJMoa2007621]
+In contrast, the authors did not find evidence of an association between exposure to an ARB at time of hospitalization and death.
+Because observational studies are subject to confounding, the greatly lower odds of death in the patients exposed to ACEis in this study cannot be interpreted as an effect of the ACEi.
```

Comment:
```suggestion
Because observational studies are subject to confounding, the reduced odds of death in the patients exposed to ACEis in this study cannot be interpreted as indicating a protective effect of the ACEi.
```